Plugins/Wox.Plugin.WebSearch/SearchSourceSetting.xaml.cs

```diff
@@ -97,10 +97,10 @@ private void AddSearchSource()
         private void EditSearchSource()
         {
             var keyword = _searchSource.ActionKeyword;
```

Comment:
nice, but this temp variable should be deleted since you have already have `newKeyword `